### PR TITLE
Add script to count unique learnersin Sensei

### PR DESCRIPTION
**Pull request by kaitohm**
Adds the script provided by @m1r0 in #2111 to wporg-learn plugin

### CODE REVIEW

**Comment by kaitohm** [comment]
@jonathanbossenger This pull request is ready for a review! 🥳 

I'll note, I suppressed a lint error in line 195:
```
$student_count = $wpdb->get_var( $query ); //phpcs:ignore WordPress.DB.PreparedSQL.NotPrepared
```
Jonathan and I discussed that this might be permissible in this case.

File: wp-content/plugins/wporg-learn/inc/sensei.php
Comment (jonathanbossenger) [inline wp-content/plugins/wporg-learn/inc/sensei.php]: @bsanevans could I ask you to indent the second and third lines of the SQL query under the first line, just to make it cleaner.

File: wp-content/plugins/wporg-learn/inc/sensei.php
Comment (jonathanbossenger) [inline wp-content/plugins/wporg-learn/inc/sensei.php:195]: @pkevan Ben and I discussed this, and we feel ignoring the PHPCS rule here is ok, but we'd appreciate your opinion. I personally can't see how this could be attacked via SQL injection, but maybe I'm missing something.

**Review by jonathanbossenger — APPROVED** [review]
This looks good to me, just waiting on feedback on that SQL line

**Comment by jonathanbossenger** [comment]
@pkevan when you have a moment, @bsanevans and I would appreciate a second pair of eyes on this PR, thanks.

File: wp-content/plugins/wporg-learn/inc/sensei.php
Comment (pkevan) [reply wp-content/plugins/wporg-learn/inc/sensei.php:195]: While it's true that the current code isn't vulnerable, that doesn't mean in the future it wouldn't be, so it's preferable to built in elements which would force it to not be.

For example, the `$from_date` variable assignment logic could be changed to grab from the `$_GET` variable directly or the appended `$query` concatenation logic changed in a way which did make it.

By adding some prepare statements, it would help avoid those scenarios, particularly when dealing with user input.

File: wp-content/plugins/wporg-learn/inc/sensei.php
Comment (kaitohm) [reply wp-content/plugins/wporg-learn/inc/sensei.php]:
```suggestion
	FROM $wpdb->comments
```

File: wp-content/plugins/wporg-learn/inc/sensei.php
Comment (kaitohm) [inline wp-content/plugins/wporg-learn/inc/sensei.php]:
```suggestion
	WHERE comment_type = 'sensei_course_status'";
```

File: wp-content/plugins/wporg-learn/inc/sensei.php
Comment (jonathanbossenger) [reply wp-content/plugins/wporg-learn/inc/sensei.php:195]: @pkevan Thanks, that's a fair point, I will work on adding sanitization to the date variables, and change the SQL concatenation to use a prepare statement.

**Comment by jonathanbossenger** [comment]
@bsanevans I have merged #2178, so I am going to close this out.

**Pull request closed without merging**
